### PR TITLE
RES-ES client tools channels

### PR DESCRIPTION
**Pull request by Loquacity**
https://bugzilla.suse.com/show_bug.cgi?id=1164723
https://github.com/SUSE/spacewalk/issues/10900

### CODE REVIEW

**Comment by Loquacity** [comment]
@srbarrios I'm not certain I understood this completely, can you please double check?

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 1]
```diff
@@ -86,15 +86,16 @@ If you do not add both channels, you will not be able to create the bootstrap re
 +
 * For {es} 6:
 +
-From the {webui}, add [systemitem]``RHEL6 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES6 x86_64``.
+From the {webui}, add [systemitem]``RHEL Expanded Support 6 x86_64``.
 +
-From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64``.
+From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem].
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc]: sth went wrong here

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 2]
```diff
@@ -86,15 +86,16 @@ If you do not add both channels, you will not be able to create the bootstrap re
 +
 * For {es} 6:
 +
-From the {webui}, add [systemitem]``RHEL6 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES6 x86_64``.
+From the {webui}, add [systemitem]``RHEL Expanded Support 6 x86_64``.
 +
-From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64``.
```
Comment (srbarrios) [inline modules/client-configuration/pages/clients-sleses.adoc]: The other two changes are exactly what I reported, but that line I'm not sure, I will need to check.
Give me some time, I will come back with it.

Comment (srbarrios) [reply modules/client-configuration/pages/clients-sleses.adoc]: As discussed during today's standup, @calancha will take care of that check.

File: modules/client-configuration/pages/clients-sleses.adoc (hunk 1 above)
Comment (calancha) [reply modules/client-configuration/pages/clients-sleses.adoc]: Line 91 was OK; it is analogous as the Expanded support 7 if we replace 7 -> 6.  That is, the original was fine:

```
From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64``.  
```

Comment (Loquacity) [inline modules/client-configuration/pages/clients-sleses.adoc]:
```suggestion
From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64``.
```

**Comment by Loquacity** [comment]
> strange mixture of webui and command prompt.

Yeah, but it's what we've used in other chapters. Otherwise, we do two separate procedures, I  guess?

**Comment by keichwa** [comment]
On 2020-03-18 11:46, Lana Brindley wrote:
>> strange mixture of webui and command prompt.
> 
> Yeah, but it's what we've used in other chapters. Otherwise, we do two
> separate procedures, I  guess?

Yes, I'd probably prefer this.  But we can improve it later.

-- 
Karl Eichwalder

**Pull request merged**